hedera-node/src/test/java/com/hedera/test/utils/ConverterUtils.java

```diff
@@ -0,0 +1,70 @@
+package com.hedera.test.utils;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import org.junit.jupiter.params.converter.ArgumentConversionException;
+
+/**
+ * Contains various common checks and methods used by the Converter classes
+ * @author abhishekpandey
+ * */
+public final class ConverterUtils {
+    /**
+     * Returns the input as a {@link String} if it is string type else throws an {@link ArgumentConversionException}
+     * @param input
+     *              the input to the converter
+     * @throws ArgumentConversionException  thrown when the input is not of type {@link String}
+     * @return input casted as a string
+     * */
+    public static String toStringInstance(final Object input)
+            throws ArgumentConversionException {
+        if (!(input instanceof String)) {
+            throw new ArgumentConversionException(input + " is not a string");
+        }
+        return (String) input;
+    }
+
+    /**
+     * Returns an array of string from the input after performing split operation else throws an
+     * {@link ArgumentConversionException}
+     * @param inputString
+     *              the input to the converter
+     * @param numberOfParts
```

Comment:
As in my other comment, I think it would read better with `exactNumberOfParts`. For the description here, we don't need, or more precisely we should not, discuss the implementation detail. So `applied for the split operation` is irrelevant. However, we should make it clear the expectation of the param, which is the exact number of parts, if it is positive, to be expected for the string returned by this function.

Reply:
Makes sense, updated to use exactNumberOfParts and the documentation.